src/main/java/parking/repositories/LotsRepositoryImpl.java

```diff
@@ -9,11 +9,13 @@
 import parking.beans.document.Account;
 import parking.beans.document.AvailablePeriod;
 import parking.beans.document.ParkingLot;
-import parking.beans.request.ParkingNumberRequest;
-import parking.beans.request.RecallSingleParking;
-import parking.beans.request.SetUnusedRequest;
 import parking.helper.ToolHelper;
+import parking.utils.ParkingType;
+import sun.util.calendar.CalendarSystem;
```

Comment:
Yes, I'm aware of warning. If we change using Date to LocalDate we would not need calendar anymore and could use Java 8 benefits. Unfortunately, this means changing the whole code because Date type was used since this app was born. 
It's a team decision, we could change to LocalDate if we agree on this and then refactor all code.